Frontend/src/components/TableEditors/Customers/Customers.svelte

```diff
@@ -4,10 +4,16 @@
   import columns from "./Columns.js";
   import filters from "./Filters.js";
   import { customerDb } from "../../../utils/stores";
+
+  const rowBackgroundColorFunction = (customer) => {
+    return customer.highlight ?? "";
+  };
 </script>
 
 <DatabaseTableEditor
   {columns}
   {filters}
+  {rowBackgroundColorFunction}
```

Comment:
```suggestion
  {(customer) => customer.highlight ?? ""}
```

Reply:
nope doesnt seem to work, syntax error, and I also think the more verbose version is easier to read and understand

Reply:
Ah sorry, it has to be `rowBackgroundColorFunction={(customer) => customer.highlight ?? ""}`. But you are right it's not really better.